packages/cozy-harvest-lib/src/models/ConnectionFlow.js

```diff
@@ -135,9 +135,10 @@ export const createOrUpdateAccount = async ({
  * This should be the go to source of truth for the state of a Konnector Job.
  */
 export class ConnectionFlow {
-  constructor(client, trigger) {
+  constructor(client, trigger, konnector) {
     this.client = client
     this.trigger = trigger
+    this.konnector = konnector
```

Comment:
Where do we use `this.konnector`?

Reply:
https://github.com/cozy/cozy-libs/blob/686036095b80a4204e6ab0bd50cfe13e9bcafbb4/packages/cozy-harvest-lib/src/models/ConnectionFlow.js#L533